### PR TITLE
Compile with Java 14

### PR DESCRIPTION
**Pull request by chbatey**
At least know our docs samples will work

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,10 @@ script: sbt -jvm-opts .jvmopts-travis -Dakka.build.scalaVersion=$TRAVIS_SCALA_VE
 
 jobs:
   include:
+    - stage: test
+      name: compile with java 14
+      script: jabba install "openjdk@~1.14.0" && jabba use openjdk@~1.14.0 && sbt -jvm-opts .jvmopts-travis -Dakka.build.scalaVersion=$TRAVIS_SCALA_VERSION "test:compile"
```
Comment (raboof) [inline .travis.yml]: instead of overwriting the script, could we set `TRAVIS_JDK=11` in this job? or are there problems with some of the other parts?

**Review by raboof — APPROVED** [review]
Nice

**Comment by johanandren** [comment]
Would it be better to have this as a nightly job than something that could potentially stop a PR from merging, given that we don't officially support 14?

**Comment by patriknw** [comment]
Shall we close this? I agree that we should start with a Jenkins in the background job, and it's not high priority as far as I know.

**Pull request closed without merging**